<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConsole</class>
 <widget class="QMainWindow" name="FormConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>控制台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCommand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>命令列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRun">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>运行命令</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCoef1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCoef1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>参数1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCoef">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>参数列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCoef2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>参数2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCoef2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDebug">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>显示调试输出</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCommand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCoef3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>参数3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCoef3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
